docs/getting_started.md

```diff
@@ -70,12 +69,12 @@ However, webKnossos includes fine-grained permissions to assign datasets to grou
 
 
 ## Your First Annotation
-To get started with your first annotation, navigate to the `Datasets` tab on your [dashboard](./dashboard.md). 
-Identify a dataset that your interested in and click on `Start Skeleton Tracing` to create a new skeleton annotation. 
-webKnossos will launch the main annotation screen allowing you to navigate your dataset and place markers to reconstruct skeletons. 
+To get started with your first annotation, navigate to the `Datasets` tab on your [dashboard](./dashboard.md).
+Identify a dataset that your interested in and click on `Start Skeleton Tracing` to create a new skeleton annotation.
```

Comment:
I know that you only changed the whitespace here, but it should be `you are` :see_no_evil: